src/components/batch-selection/batch-selection.component.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import I18n from 'i18n-js';
+import PropTypes from 'prop-types';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import { StyledBatchSelection, StyledSelectionCount } from './batch-selection.style';
+
+const BatchSelection = ({
+  disabled,
+  children,
+  colorTheme,
+  selectedCount,
+  hidden
+}) => {
+  return (
+    <StyledBatchSelection
+      colorTheme={ colorTheme }
+      data-component='batch-selection'
+      disabled={ disabled }
+      hidden={ hidden }
+    >
+      <StyledSelectionCount data-element='selection-count'>
+        <span>{ selectedCount }</span> { I18n.t('selected') }
+      </StyledSelectionCount>
+      { children }
+    </StyledBatchSelection>
+  );
+};
+
+BatchSelection.propTypes = {
+  /** Color of the background, transparent if not defined */
+  colorTheme: PropTypes.oneOf(OptionsHelper.flatTableThemes),
+  /** Content to be rendered after selected count */
+  children: PropTypes.node.isRequired,
```

Comment:
Can you put `isRequired` props first? It will be better for the end user

Reply:
good point, done